packages/react-refresh-utils/ReactRefreshWebpackPlugin.ts

```diff
@@ -95,6 +95,7 @@ function webpack5(compiler: Compiler) {
       // @ts-ignore This exists in webpack 5
       const { runtimeTemplate } = this.compilation
       return Template.asString([
+        `if (${RuntimeGlobals.interceptModuleExecution}) {`,
```

Comment:
Is there a specific integration test to be added for this case? It's unclear what it solves.

Reply:
React refresh template is injected in web worker bundle and an exception is created: `Uncaught TypeError: Cannot read property 'push' of undefined`

Here there is a repro https://github.com/ramasilveyra/bug-repro-next-w5-worker

I will update pr description with this info.

The exception happens when web worker is executed, so the current worker loader test covers this case.